vispy/scene/visuals.py

```diff
@@ -3,7 +3,9 @@
 # Distributed under the (new) BSD License. See LICENSE.txt for more info.
 """
 The classes in scene.visuals are visuals that may be added to a scenegraph
-using the methods and properties defined by vispy.scene.Node. 
+using the methods and properties defined in vispy.scene.Node such as name,
+visible, parent, children, etc...
```

Comment:
It would be great if we can figure out how to make these show up in the documentation for each individual VisualNode class.